main_fixed.py

```
@@ -0,0 +1,378 @@
+                await callback.answer("База данных недоступна", show_alert=True)
+                return
+                
+            from src.database.user_operations import get_account_info
+            accounts = await get_account_info(pool, callback.from_user.id)
+            
+            if not accounts:
+                text = "❌ У вас нет зарегистрированного аккаунта.\n\n" \
+                       "Используйте кнопку 'Регистрация' для создания аккаунта."
+            else:
+                text = "🔑 Ваши аккаунты:\n\n"
+                for i, (email, username, is_temp, temp_password) in enumerate(accounts, 1):
```

Ruff (0.12.2)

Loop control variable `temp_password` not used within loop body

Rename unused `temp_password` to `_temp_password`

(B007)